llvm/lib/Target/AIE/AIE2AliasAnalysis.cpp

```diff
@@ -0,0 +1,68 @@
+//===- AIE2AliasAnalysis.cpp ----------------------------------------------===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+//
+//===----------------------------------------------------------------------===//
+/// \file
+/// This is the AIE alias analysis pass. It looks through AIE's complex
```

Comment:
We need to adapt this comments.

Reply:
Please suggest what part of it should be adapted/updated. Thanks

Reply:
The comment says that we look through AIE's complex addressing modes. However we look to banks here.